### PR TITLE
Pin the APM libraries to major versions

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
This will pin users copying the commands to the latest major version of the libraries. This will give the user automatic fixes and features without the risk of breaking changes.

[](https://datadoghq.atlassian.net/browse/APMON-1258)

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by avivenzio-dd — COMMENTED** [review]
just wanted to ask if we wanted to wait until front end feature flag is on in all DCs before merging this?

IMO I think it is ok to not wait as this is all script updates and those will work regardless of ui

**Review by jhgilbert — APPROVED** [review]
I see a query around whether this should be merged -- @Kyle-Verhoog, can you confirm that you would like me to merge it? I'll just leave it as approved for now. Thank you!

File: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md
Comment (alexbryukhanova) [inline content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md:140]: I might be spacing out but what was the actual consensus on the whole `v` prefix?

**Review by vidurkhannadatadog — COMMENTED** [review]
Can we have a small blob for major version pinning and how to use them, the way we have it for regular versions here:

>  For example, to install support for only v1.25.0 of the Java tracing library and the latest Python tracing library, add the following to the installation command: ... `

File: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md
Comment (Kyle-Verhoog) [reply content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md:140]: ah good catch. We can safely drop the v-prefix now.

**Review by vidurkhannadatadog — APPROVED** [review]
LGTM.

**Comment by Kyle-Verhoog** [comment]
@jhgilbert could take a look again? I made some changes since the last review. It is good to merge once we have your approval. Thanks!!

**Review by jhgilbert — APPROVED** [review]

**Pull request merged**
